.github/workflows/pythonpackage.yml

```diff
@@ -4,7 +4,10 @@
 name: Python package
 
 on:
-  [push, pull_request]
+  push:
+  pull_request:
+  schedule:
+    - cron: '25 1 3 * *'
```

Comment:
I would recommend something like `'25 1 * * 3'` - as that is disabled after 3 months, so would run around only 2 times. If you run it weekly, you will get a run closer to when the cron CI is disabled